MediaWikiKit/MediaWikiKit/MWKSectionList.m

```diff
@@ -140,4 +140,37 @@ - (NSArray*)entries {
     return self.sections;
 }
 
+#pragma mark - Hierarchy
+
+- (void)buildSectionHierarchy {
```

Comment:
I thought about that... but figured we only need it in one place right now, and can revisit if/when anywhere else needs hierarchical data about sections (YAGNI). I was reluctant to even make `children` a property on `MWKSection`, but this goes with the rest of MWK in WYSINAWYG (what you see is not always what you get)